cfgov/cfgov/settings/base.py

```diff
@@ -154,6 +154,7 @@
         'OPTIONS': {
             'environment': 'v1.environment',
             'extensions': [
+                'v1.jinja2tags.images',
```

Comment:
:+1: for encapsulating Jinja tags into separate modules instead of dumping into the `v1.environment` list.